examples/guide/frontend/guide-with-facet-groups.json

```diff
@@ -63,6 +63,8 @@
           "facet_group": [
             {
               "content_id": "52435175-82ed-4a04-adef-74c0199d0f46",
+              "base_path": "/blah",
```

Comment:
Facet groups do not have base paths, so this example isn't representative of the link as it appears in the content store response.
I think the problem lies with how frontend schemas are generated, they don't cater for pathless links like publishing schema generation does. See https://github.com/alphagov/govuk-content-schemas/pull/884 for context. It might be better to fix this rather than create an unrealistic example.
I also don't understand the need for this example in this PR, is this example necessary to represent the link to a finder in the content item's `links["finder"]` guid list?

Reply:
You're right, the `facet_group` isn't needed, but it does make for a more complete example when the `finder` is added to the links.

Reply:
Thanks for the link to the fix! Implementing it now.